internal/studio/oas_studio.yaml

```diff
@@ -43,27 +34,10 @@ paths:
         content:
           application/json:
             schema:
-              type: object
-              properties:
-                overlay:
-                  type: string
-                  description: The studio modifications overlay contents - this should be an overlay YAML document
-                input:
-                  type: string
-                  description: The input spec for the source
-                targets:
-                  type: object
-                  additionalProperties:
-                    $ref: "#/components/schemas/TargetSpecificInputs"
-                  description: Map of target specific inputs keyed on target name
+                $ref: "#/components/schemas/RunRequestBody"
       responses:
         "200":
-          description: Successful response
-          content:
-            text/event-stream:
-              schema:
-                oneOf:
-                  - $ref: "#/components/schemas/RunResponseStreamEvent"
```

Comment:
@mfbx9da4 were we expecting to expand the  `oneOf` here? note I have removed it while refactoring